index.js

```diff
@@ -49,8 +50,12 @@ const serveClient = async () => {
   const nextDistroEnrollmentForm = require('./client/utils/data.js')
   try {
     await fullStack.get('/', (req, res) => {
-      nextDistroEnrollmentForm.then( data => res.status(200).render('index', {data}))
+      nextDistroEnrollmentForm.data.then( (data) => res.status(200).render('index', {data}))
```

Comment:
@cofuente It didn't work without the data variable.

Reply:
@cofuente `nextDistroEnrollmentForm.then is not a function`

Reply:
Yes, of course. But I was just pointing out that that's only because in [data.js](https://github.com/cofuente/postgres-express-backend/blob/12c5a98a1e84ba470d54a44116257d188f99728c/client/utils/data.js#L12), you export it as data in the first place. Which I just thought was strange. Anyway, doesn't really matter. Greenlit!